addons/LDtk-Importer/LDtk.gd

```diff
@@ -232,18 +232,25 @@ func tileId_to_pxCoords(tileId, atlasGridSize, atlasGridWidth, padding, spacing)
 
 	return Vector2(pixelTileX, pixelTileY)
 
+
 func import_collisions(tilemap_data, options):
 #return if options has collision import off or layer name isn't "Collisions"
-	var shouldImportCollisions = options.Import_Collisions and tilemap_data.__identifier == "Collisions"
-	if not shouldImportCollisions:
-		return
+	if tilemap_data.__identifier == "Collisions":
+		var Collisions = options.Import_Collisions and tilemap_data.__identifier == "Collisions"
+		if not Collisions:
+			return
+
```

Comment:
I sometimes add randomly high amounts of blank lines. You might want to remove it.